saltproc/app.py

```diff
@@ -264,47 +276,58 @@ def _create_simulation_object(simulation_input, depcode, cores, nodes):
 
 def _create_reactor_object(reactor_input):
     """Helper function for `run()` """
-    msr = Reactor(
-        volume=reactor_input['volume'],
-        mass_flowrate=reactor_input['mass_flowrate'],
-        power_levels=reactor_input['power_levels'],
-        dep_step_length_cumulative=reactor_input['dep_step_length_cumulative'])
+    msr = Reactor(**reactor_input)
     return msr
 
 
-def _process_main_input_reactor_params(reactor_input, num_depsteps):
+def _process_main_input_reactor_params(reactor_input, n_depletion_steps, codename):
     """
     Process SaltProc reactor class input parameters based on the value and
-    data type of the `num_depsteps` parameter, and throw errors if the input
+    data type of the `n_depletion_steps` parameter, and throw errors if the input
     parameters are incorrect.
```

Comment:
Consider mentioning the added `codename` parameter in this docstring

Reply:
done